examples/test/AttributeFilter_test.js

```diff
@@ -36,6 +37,43 @@ test("Dropdown filter opens, clears, selects all and applies", async t => {
         .notOk();
 });
 
+test("positive, negative amd error attribute filters", async t => {
+    const dropdownButton = Selector(".s-location_resort.button");
+    const lineChart = Selector(".s-line-chart");
+    const markers = Selector(".highcharts-markers path");
+    const error = Selector(".s-error .info-label");
+    await t
```

Comment:
Maybe broke this to groups, i.e., make `await.t` chain after each `expect` and write an error message (a reason for failure) into each `expect` condition (or at least to some, for example to ones that have an only mysterious number in it) to improve readability.

Example:

```
.expect(markers.count)
.eql(2)
```

.expect(markers.count)
.eql(2, "Line chart displays more markers than expected") // or something else, I'm not sure what this test does which is the reason why it should be made more readable